worldedit-bukkit/adapters/adapter-1_20_2/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_20_R2/PaperweightGetBlocks.java

```diff
@@ -383,6 +386,51 @@ public Iterator<CompoundTag> iterator() {
         };
     }
 
+    @Override
+    public Set<com.sk89q.worldedit.entity.Entity> getFullEntities() {
+        List<Entity> entities = PaperweightPlatformAdapter.getEntities(ensureLoaded(serverLevel, chunkX, chunkZ));
+        if (entities.isEmpty()) {
+            return Collections.emptySet();
+        }
+        int size = entities.size();
+        return new AbstractSet<>() {
+            @Override
+            public int size() {
+                return size;
+            }
+
+            @Override
+            public boolean isEmpty() {
+                return false;
+            }
+
+            @Override
+            public boolean contains(Object get) {
+                if (!(get instanceof com.sk89q.worldedit.entity.Entity e)) {
+                    return false;
+                }
+                UUID getUUID = e.getState().getNbtData().getUUID();
+                for (Entity entity : entities) {
+                    UUID uuid = entity.getUUID();
+                    if (uuid.equals(getUUID)) {
+                        return true;
+                    }
+                }
+                return false;
+            }
+
+            @Nonnull
+            @Override
+            public Iterator<com.sk89q.worldedit.entity.Entity> iterator() {
+                Iterable<com.sk89q.worldedit.entity.Entity> result = entities
+                        .stream()
+                        .map(input -> new BukkitEntity(input.getBukkitEntity()))
+                        .collect(Collectors.toList());
```

Comment:
It should be possible to return `Stream#iterator()` here directly, avoiding some memory overhead